### DISCUSSION TITLE
Make the Nutri-Score detection prediction actionable ("Delete" button & Thumbnail)

### DISCUSSION THREAD

**Discussion by teolemon** [openfoodfacts/robotoff#1240] [Ideas]
### What

- [ ] Show source image in Nutri-Score detection prediction notifications on Slack 
- [ ] Add a 1-click "Remove value" button
- [ ] Add an "Edit product" button
### Why
- Those predictions are applied automatically, and it should be easy to check them, and to rectify errors)

### How
- Logo notifications (and others) already include the source image, so code reuse should be possible.
- Code pointer: https://github.com/openfoodfacts/robotoff/blob/master/robotoff/slack.py

### Related
- #115 
- #355 

### Mockup

![image](https://user-images.githubusercontent.com/1689815/131979189-a4b4bb3d-35d9-42de-b34e-9e2fb3cd3d68.png)

**Comment by alexgarel** [comment]
@Jagrutiti: you should first try to  _Add an "Edit product" button_ as it's a simple url to build: https://world.openfoodfacts.org/cgi/product.pl?type=edit&code=<code_goes_here>

**Comment by alexgarel** [comment]
For image, you should get the crop, see `LogoAnnotation.get_crop_image_url`

**Comment by alexgarel** [comment]
As said on slack:

you will have to go before notify_automatic_processing, at import processus to try to put the right source_image to the insight

Look at extract_nutriscore_label in robotoff.insights.extraction you should:
1. add the source image as parameter to extract_nutriscore_label, and then add it to the Prediction as source_image
2. extract LogoAnnotation.get_crop_image_url  as a crop_image_url function in models, and make LogoAnnotation.get_crop_image_url  just call it with right parameter.
3. in slack notification function, when there is a source_image, if there is a bounding_box in the data parameter, use it to build an url using crop_image_url (and keeping actual behaviour if there is none)

**Comment by raphael0202** [comment]
Is this issue fixed?
